platform/configure/config.mdx

```diff
@@ -1,20 +1,24 @@
 ---
-title: vCluster Platform Configuration
-sidebar_label: vCluster Platform Config
+title: Platform Configuration
+sidebar_label: Platform Config
 sidebar_position: 1
+description: Learn how to configure the vCluster Platform, modify its behavior, and enable or disable features such as SSO login, custom branding, and auditing.
 ---
 
 import VClusterProConfig from "../api/_partials/resources/config/status_reference.mdx";
 
-The vCluster Platform config allows you to modify vCluster Platform's behaviour and enable or disable certain features, such as SSO login, custom branding, or auditing.
+Modify the platform's behavior and enable or turn off certain features, such as SSO login, custom branding, or auditing.
```

Comment:
This should go into more detail about how the `config` is a secation of part of the values.yaml that details out how the platform runs.